composer/utils/object_store/s3_object_store.py

```diff
@@ -79,11 +79,12 @@ def __init__(
     ) -> None:
         try:
             import boto3
-            from boto3.s3.transfer import TransferConfig
+            from boto3.s3.transfer import S3Transfer, TransferConfig
             from botocore.config import Config
         except ImportError as e:
             raise MissingConditionalImportError('streaming', 'boto3') from e
 
+        self.ALLOWED_UPLOAD_ARGS = S3Transfer.ALLOWED_UPLOAD_ARGS
```

Comment:
why does this need to be saved? can we just always use `S3Transfer.ALLOWED_UPLOAD_ARGS`